apps/gallery/style/gallery.css

```diff
@@ -194,6 +194,15 @@ body.pickView #thumbnails {
   top: 5rem;
 }
 
+/*
+ * Give thumbnails container a large bottom margin so that it doesn't
+ * hide behind the overlaid toolbar in thumbnail list and select view.
+ */
+body.thumbnailSelectView #thumbnails,
+body.thumbnailListView #thumbnails {
+  margin-bottom: 4rem;
```

Comment:
You have 4.5 in the old patch. Shouldn't you have that here, too? Or is there and extra .5 rem added from somewhere else?